src/main/resources/init_db.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE `ma_db`.`books` (
+    `id` INT NOT NULL AUTO_INCREMENT,
+    `title` VARCHAR(255) NULL,
+    `price` DECIMAL(8,2) NULL,
+    PRIMARY KEY (`id`)
+);
```

Comment:
make sure each file in your code ends with exactly one empty line
![image](https://github.com/mate-academy/jv-jdbc-intro/assets/49767049/ed07e1cc-78c1-44c8-a292-2a759d88b5f3)